src/app/client/src/app/app.component.ts

```diff
@@ -111,14 +111,33 @@ export class AppComponent implements OnInit {
         this.tenantService.getTenantInfo(this.slug);
         this.setPortalTitleLogo();
         this.telemetryService.initialize(this.getTelemetryContext());
+        this.logCdnStatus();
         this.checkTncAndFrameWorkSelected();
         this.initApp = true;
       }, error => {
         this.initApp = true;
       });
     this.changeLanguageAttribute();
   }
-
+  logCdnStatus() {
+    const isCdnWorking  = (<HTMLInputElement>document.getElementById('cdnWorking'))
+    ? (<HTMLInputElement>document.getElementById('cdnWorking')).value : 'no';
+    if (isCdnWorking !== 'no') {
```

Comment:
String.toLowerCase() is missing here. Always make sure when string comparison is happening then toLowerCase() is must